azkaban-common/src/main/java/azkaban/project/JdbcProjectLoader.java

```diff
@@ -193,17 +193,23 @@ private Project fetchProjectByName(Connection connection, String name)
         throws ProjectManagerException {
         QueryRunner runner = new QueryRunner();
         // Fetch the project
-        Project project = null;
+        Project project;
```

Comment:
Intellij gives this warning saying variable 'project' initializer null is redundant.  It will be assigned value later in project = projects.get(0); It there is any exception, then we don't even need to return project.